dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/LoggerService.java

```diff
@@ -75,7 +75,17 @@ public Result queryLog(int taskInstId, int skipLineNum, int limit) {
 
     logger.info("log host : {} , logPath : {} , logServer port : {}",host,taskInstance.getLogPath(),Constants.RPC_PORT);
 
-    String log = logClient.rollViewLog(host, Constants.RPC_PORT, taskInstance.getLogPath(),skipLineNum,limit);
+    StringBuilder log = new StringBuilder();
+    if(skipLineNum == 0 ) {
```

Comment:
```suggestion
    StringBuilder log = new StringBuilder();
    if (skipLineNum == 0) {
```
Hi,

Good job and thx a lot for your contribution, please configure the `checkstyle.xml`[1] to IDE and it will help to format code.

[1] https://github.com/apache/incubator-dolphinscheduler/tree/dev/style